tests/test_modeling_utils.py

```diff
@@ -15,6 +15,7 @@
 
 import inspect
 import math
+import pdb
```

Comment:
we should be careful to always remove those before merging :-) Totally fine to keep them for testing though!